tests/network.md

```diff
@@ -485,3 +485,24 @@ EPIPE:
     (fun () -> Eio.Flow.shutdown a `Receive);;
 - : unit = ()
 ```
+
+## Addrinfo (Linx and Libuv)
+
+```ocaml
+# Eio_main.run @@ fun env ->
+  Eio.Net.addrinfo env#net "localhost";;
+- : ([ `Tcp | `Udp ] * Eio.Net.Ipaddr.v4v6) list =
+[(`Tcp, "\000\000\000\000\000\000\000\000\000\000\000\000\000\000\000\001");
+ (`Udp, "\000\000\000\000\000\000\000\000\000\000\000\000\000\000\000\001");
+ (`Tcp, "\127\000\000\001"); (`Udp, "\127\000\000\001")]
+```
+
+<!-- $MDX set-EIO_BACKEND=luv -->
```

Comment:
We need a proper fix for this (at the moment, I run `make test_luv` to run all the tests this way). Probably we should have a `tests/luv` directory with some dune magic to duplicate all the tests.

Reply:
Removed luv test and using `make test_luv` instead.